hack/docs/Adding_Detectors_external.md

```diff
@@ -62,15 +62,29 @@ If you think that something should be included outside of these guidelines, plea
    1. Create a [test secrets file, and export the variable](#using-a-test-secret-file)
    2. Update the pattern regex and keywords. Try iterating with [regex101.com](http://regex101.com/).
    3. Update the verifier code to use a non-destructive API call that can determine whether the secret is valid or not.
-   4. Update the tests with these test cases at minimum:
+      * Make sure you understand [verification indeterminacy](#verification-indeterminacy).
+   5. Update the tests with these test cases at minimum:
```

Comment:
__nit__: This should be `4.`. Though it should render properly regardless.

Reply:
i wanted to make them all `1.` but goland yelled at me :(